bundles/org.eclipse.equinox.launcher/src/org/eclipse/equinox/launcher/Main.java

```diff
@@ -1089,7 +1086,7 @@ protected String searchFor(final String target, String start) {
 		if (result == -1) {
 			return null;
 		}
-		File candidate = new File(start, names[result]);
+		File candidate = new File(root, names[result]);
```

Comment:
@tjwatson is it actually intended here to return the unresolved `start` directory?

Reply:
I don't think so, that does look wrong to me.  Particularly because we then try to get the absolute path to the unresolved start as the returned result.  How is this working now?

Reply:
My guess is that the `start` file is actually never resolved because it's absolute.
But I haven't tried to reproduce it.